gen.py

```diff
@@ -55,15 +57,17 @@
 
 # User selects the theme
 if not args.theme:
-    exit("Please use --theme argument with "
-         "one of the following: {}".format(", ".join(themes)))
+    msg = "Please use --theme argument with one of the following: "
+    exit(msg + ", ".join(themes))
```

Comment:
There were quite a few cases where `format` wasn't needed; standard string concatenation sufficed so uses that.